rmgpy/rmg/react.py

```diff
@@ -104,6 +104,112 @@ def reactMolecules(moleculeTuples):
     return reactionList
 
 def deflate(rxns, molecules, reactantIndices):
+def findDegeneracies(rxnList, useSpeciesReaction = True):
+    """
+    given a list of reaction object with Molecule objects, this method 
+    removes degenerate reactions and
+    increments the degeneracy of the reaction object. This method modifies
+    rxnList in place and does not return anything.
+
+    This algorithm used to exist in family.__generateReactions, but was moved
+    here because it didn't have any family dependence.
+    """
+    # These duplicates should be combined and the reaction degeneracy should be increased
+    # Reaction degeneracy is only increased if two reaction are isomorphic but resulted
+    # from different transition states.
```

Comment:
This was my comment from earlier changes, before we decided to tackle multiple transition states. In this comment, different transition states refers to ones that are the effectively identical but involving a different set of atoms, which i guess you could also call degenerate transition states.

It should probably be changed now that we're also dealing with actually different transition states.